code/__DEFINES/antagonists.dm

```diff
@@ -74,8 +74,10 @@
 
 ///It is faster as a macro than a proc
 #define IS_HERETIC(mob) (mob.mind?.has_antag_datum(/datum/antagonist/heretic))
-#define IS_HERETIC_MONSTER(mob) (mob.mind?.has_antag_datum(/datum/antagonist/heretic_monster))
+#define IS_HERETIC_CULTIST(mob) (mob.mind?.has_antag_datum(/datum/antagonist/heretic_monster))
 
+GLOBAL_LIST_INIT(heretic_start_knowledge,list(/datum/eldritch_knowledge/codex_cicatrix,/datum/eldritch_knowledge/living_heart,/datum/eldritch_knowledge/spell/basic,/datum/eldritch_knowledge/convert,/datum/eldritch_knowledge/eldritch_avatar))
```

Comment:
Could this not be a define? It has no need to ever be changed.

Reply:
Oh. It was like this in the updated tg version.